### PR TITLE
Opportunity asserts

### PR DESCRIPTION
**Pull request by Paola-Xaq**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    @FindBy(name = "SaveAndNew")
+    private WebElement saveAndNewBtn;
+
+    @FindBy(name = "CancelEdit")
+    private WebElement cancelBtn;
+
+    private static final String XPATH_INPUT = "//input[@name='%s']";
+    private static final String XPATH_DROPDOWN_ELEMENT = "//label[text()='%s']/..//input";
+    private static final String XPATH_DROPDOWN_OPTION = "//lightning-base-combobox-item[@data-value='%s']";
+    //private static final String XPATH_SEARCH = "//input[contains(@placeholder,'%s')]";
```
Comment (JRojas999) [inline src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java]: If these won't be used it should be deleted

**Comment by JRojas999** [comment]
It should have checkstyle plugin.

**Review by JRojas999 — APPROVED** [review]

File: src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunity.java [hunk 2]
```diff
@@ -1,10 +1,108 @@
...
+    /**
+     * Clicks the detail button for created Oppoprtunity.
+     */
+    public void clickDetails() {
+        webElementAction.clickBtn(driver.findElement(By.xpath("//a[@data-tab-value='detailTab']")));
+        //NEEDED TO WAIT FOR JAVASCRIPT TO LOAD
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException ie) {
+
```
Comment (cristianfchq) [inline src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunity.java]: This line chould be removed

File: src/test/java/salesforce/opportunity/OpportunityTests.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        String opportunityName = "New opportunity name";
+        String opportunityCloseDate = "7/31/2021";
+        String opportunityStage = "Prospecting";
+
+        formOpportunity = opportunityPage.openNewOpportunityForm();
+        formOpportunity.setDropdown("Stage", opportunityStage);
+        formOpportunity.setInputField("Name", opportunityName);
+        formOpportunity.setCloseDate(opportunityCloseDate);
+        createdForm = formOpportunity.clickSaveOpportunity();
+        //Assert with Alert when creating
```
Comment (cristianfchq) [inline src/test/java/salesforce/opportunity/OpportunityTests.java:29]: this comments should be removed

File: src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java (hunk 1 above)
Comment (Paola-Xaq) [reply src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java]: ok, it's done, thanks :)

File: src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunity.java (hunk 2 above)
Comment (Paola-Xaq) [reply src/main/java/salesforce/ui/pages/opportunity/CreatedOpportunity.java]: ok, it's done, thanks :)

File: src/test/java/salesforce/opportunity/OpportunityTests.java (hunk 3 above)
Comment (Paola-Xaq) [reply src/test/java/salesforce/opportunity/OpportunityTests.java:29]: These comments are just there to sepárele the different asserts, but I will remove it in the next PR

**Pull request merged**